src/SumProc.jl

```diff
@@ -107,36 +108,45 @@ function nesum(a::CycloFrac{T}, var::Int64, lower::Int64, upper::Union{Int64,T},
 		sum, exc = nesum(a, var, 0, upper, congruence)
 		return (sum-nesum(a, var, 0, lower-1), exc)
```

Comment:
Unrelated to this PR, but:

Is this code actually correct? Wouldn't the `nesum` call produce two return values, and so we get an error here? And what about the `congruence` argument?

How about this:
```suggestion
        sum2, exc2 = nesum(a, var, 0, lower-1, congruence)
		return sum-sum2, union(exc, exc2)
```

Perhaps not in this PR but in another, and perhaps you could add a test case or two for this?

Reply:
Oh, you are correct. This wouldn't work.